server/.env

```diff
@@ -0,0 +1 @@
+DBConnLink = "postgres://project_data_eqrf_user:MldQZenHLTFNPhwOUA6Nx6a9zrKRMfXc@dpg-ck1o4bnhdsdc739osv00-a.oregon-postgres.render.com/project_data_eqrf";
```

Comment:
You probably didn't mean to check this file in - you should probably remove it from git, change the password, and add it to your .gitignore

Reply:
i fixed it